SPECS/asciidoc/asciidoc.spec

```diff
@@ -41,23 +39,24 @@ rm -rf %{buildroot}%{_infodir}
 make DESTDIR=%{buildroot} install
 
 %check
-python tests/testasciidoc.py update
-python tests/testasciidoc.py run
+python3 tests/testasciidoc.py update
```

Comment:
Nit: We could use the `%python3` macro here, [since we have it.](https://github.com/microsoft/CBL-Mariner/blob/1f0a8ae9fa285384aee0465a1e2aa7b9c0ec4ed4/SPECS/mariner-rpm-macros/macros.python-srpm#L12)

Reply:
I'm going to defer the macro nits to later work, since this is a huge commit as is, and I'm worried about making more changes that require rebuilds to test.